### PR TITLE
Delete submission errors every attempt to upload

### PR DESCRIPTION
**Pull request by aaclan-ebi**
ebi-ait/dcp-ingest-central#13

### CODE REVIEW

File: ingest/importer/importer.py [hunk 1]
```diff
@@ -61,6 +62,7 @@ def import_file(self, file_path, submission_url, is_update=False, project_uuid=N
             template_mgr = None
             spreadsheet_json, template_mgr, errors = self.generate_json(file_path, is_update, project_uuid=project_uuid)
             entity_map = EntityMap.load(spreadsheet_json)
+            self.ingest_api.delete_submission_errors(submission_url)
```
Comment (ke4) [inline ingest/importer/importer.py:65]: Well, I am not sure this call belongs to here. Shall we move it to the broker?

**Review by ke4 — CHANGES_REQUESTED** [review]
Added a question

File: ingest/importer/importer.py (hunk 1 above)
Comment (aaclan-ebi) [reply ingest/importer/importer.py:65]: why broker?

Comment (aaclan-ebi) [reply ingest/importer/importer.py:65]: The method report_errors is being called in this class, if I'm going to move the delete_submission_errors to broker, we should also move that.

Comment (ke4) [reply ingest/importer/importer.py:65]: Let's leave it like this for now as time pressure, but would be nice to refactor this later as this method is doing too much.

**Review by ke4 — APPROVED** [review]

**Comment by aaclan-ebi** [comment]
We only have @ke4 available today. Merging this before I go on holiday.

**Pull request merged**